spec/shared_context.rb

```diff
@@ -39,6 +39,14 @@
   end
 end
 
+RSpec.shared_context 'shared context Oneview API 600', a: :b do
+  before :each do
+    api_version = 600
```

Comment:
not sure if you really need shared context if you are declaring API version in the unit test cases